js/modules/venn.src.js

```diff
@@ -434,14 +302,56 @@ var getLabelPosition = function getLabelPosition(internal, external) {
 };
 
 /**
- * Calulates data label positions for a list of relations.
+ * Finds the available width for a label, by taking the label position and
+ * finding the largest distance, which is inside all internal circles, and
+ * outside all external circles.
+ *
+ * @private
+ * @todo Add unit tests
```

Comment:
Is this still a `@todo`? There are now unit tests below...

Reply:
It was still to be done, but I took the time to add them now to the PR.